enterprise_access/apps/bffs/handlers.py

```diff
@@ -216,14 +216,19 @@ def transform_subscriptions_result(self, subscriptions_result):
             'subscription_licenses_by_status': subscription_licenses_by_status,
         }
 
-    def check_has_activated_license(self):
+    @property
+    def current_active_license(self):
```

Comment:
Good call to move this to a `@property` :) Been thinking it might make sense to do that.